src/worker.rs

```diff
@@ -241,7 +241,10 @@ impl<T: Socket + ?Sized> Worker<T> {
     }
 
     fn send_packet(&self, packet: &Packet) -> Result<(), Box<dyn Error>> {
-        for _ in 0..self.duplicate_packets {
+        for i in 0..self.duplicate_packets {
+            if i > 0 {
```

Comment:
This is good, but is the intended effect putting the delay only between the packets, or before each packet?

Reply:
The delay is between the Original and the Duplicate Packets. I tested adding the delay between the Original Packets, but it doesn't fix the TFTP Timeouts.